project/src/test/resources/docker/sql/insert-essential-data.sql

```diff
@@ -0,0 +1,293 @@
+('my'),
+('ca'),
+('ch'),
+('ce'),
+('ny'),
+('zh'),
+('zh-Hans'),
+('zh-Hant'),
+('cv'),
+('kw'),
+('co'),
+('cr'),
+('hr'),
+('cs'),
+('da'),
+('dv'),
+('nl'),
+('dz'),
+('eo'),
+('et'),
+('ee'),
+('fo'),
+('fj'),
+('fi'),
+('fr'),
+('ff'),
+('gl'),
+('gd'),
+('gv'),
+('ka'),
+('de'),
+('el'),
+('kl'),
+('gn'),
+('gu'),
+('ht'),
+('ha'),
+('he'),
+('hz'),
+('hi'),
+('ho'),
+('hu'),
+('is'),
+('io'),
+('ig'),
+('id'),
+('in'),
+('ia'),
+('ie'),
+('iu'),
+('ik'),
+('ga'),
+('it'),
+('ja'),
+('jv'),
+('kn'),
+('kr'),
+('ks'),
+('kk'),
+('km'),
+('ki'),
+('rw'),
+('rn'),
+('ky'),
+('kv'),
+('kg'),
+('ko'),
+('ku'),
+('kj'),
+('lo'),
+('la'),
+('lv'),
+('li'),
+('ln'),
+('lt'),
+('lu'),
+('lg'),
+('lb'),
+('mk'),
+('mg'),
+('ms'),
+('ml'),
+('mt'),
+('mi'),
+('mr'),
+('mh'),
+('mo'),
+('mn'),
+('na'),
+('nv'),
+('ng'),
+('nd'),
+('ne'),
+('no'),
+('nb'),
+('nn'),
+('oc'),
+('oj'),
+('cu'),
+('or'),
+('om'),
+('os'),
+('pi'),
+('ps'),
+('fa'),
+('pl'),
+('pa'),
+('qu'),
+('rm'),
+('ro'),
+('ru'),
+('se'),
+('sm'),
+('sg'),
+('sa'),
+('sr'),
+('sh'),
+('st'),
+('tn'),
+('sn'),
+('ii'),
+('sd'),
+('si'),
+('sk'),
+('sl'),
+('so'),
+('nr'),
+('es'),
+('su'),
+('sw'),
+('ss'),
+('sv'),
+('tl'),
+('ty'),
+('tg'),
+('ta'),
+('tt'),
+('te'),
+('th'),
+('bo'),
+('ti'),
+('to'),
+('ts'),
+('tr'),
+('tk'),
+('tw'),
+('ug'),
+('uk'),
+('ur'),
+('uz'),
+('ve'),
+('vi'),
+('vo'),
+('wa'),
+('cy'),
+('wo'),
+('fy'),
+('xh'),
+('yi'),
+('ji'),
+('yo'),
+('za')
+ON CONFLICT DO NOTHING;
+
+INSERT INTO dbo.ICalendarDataType(name) VALUES
+('BINARY'),
+('BOOLEAN'),
+('CAL-ADDRESS'),
+('DATE'),
+('DATE-TIME'),
+('DURATION')
+ON CONFLICT DO NOTHING;
+
+INSERT INTO dbo.Category OVERRIDING SYSTEM VALUE VALUES(1) ON CONFLICT DO NOTHING; -- 1
+INSERT INTO dbo.Category OVERRIDING SYSTEM VALUE VALUES(2) ON CONFLICT DO NOTHING; -- 2
+INSERT INTO dbo.Category OVERRIDING SYSTEM VALUE VALUES(3) ON CONFLICT DO NOTHING; -- 3
+INSERT INTO dbo.Category OVERRIDING SYSTEM VALUE VALUES(4) ON CONFLICT DO NOTHING; -- 4
+INSERT INTO dbo.Category OVERRIDING SYSTEM VALUE VALUES(5) ON CONFLICT DO NOTHING; -- 5
+INSERT INTO dbo.Category OVERRIDING SYSTEM VALUE VALUES(6) ON CONFLICT DO NOTHING; -- 6
+
+INSERT INTO dbo.CategoryLanguage(category, name, language) VALUES
+(1, 'Exame',       1),
+(2, 'Aula',        1),
+(3, 'Laboratório', 1),
+(4, 'Aviso',       1),
+(5, 'Entrega',     1),
+(6, 'Teste',       1),
+(1, 'Exam',        2),
+(1, 'Exam',        3),
+(2, 'Lecture',     2),
+(2, 'Lecture',     3),
+(3, 'Laboratory',  2),
+(3, 'Laboratory',  3),
+(4, 'Warning',     3),
+(4, 'Warning',     2),
+(5, 'Deadline',    2),
+(5, 'Deadline',    3),
+(6, 'Test',        2),
+(6, 'Test',        3)
+ON CONFLICT DO NOTHING;
+
+
+-- AM
+-- read:calendar scope only used by derived tokens
+INSERT INTO dbo.scopes (scope) VALUES
+('urn:org:ionproject:scopes:api:read'),
+('urn:org:ionproject:scopes:api:write'),
+('urn:org:ionproject:scopes:token:issue'),
+('urn:org:ionproject:scopes:api:read_restricted'),
```

Comment:
this token is only used for testing, its not essential for the production environment